spec/models/user_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
```

Comment:
Great job @wolterbwambale 🤝

- [ ] As stated in the project requirements, kindly ensure the User model has validation tests for the name must not be blank.